components/test-page-setup.js

```diff
@@ -207,32 +233,63 @@ export function makeTwoOutputTest(testParams, testName, testDescription, shaderC
             </div>
             <div className="columns is-one-fifth">
               <div className="column">
-                    <Bar
-                      data={chartData(testState)}
-                      options={chartConfig(pageState)}
-                    />
+                  {clearChart 
+                   ? 
+                   <Bar
+                    data={chartDataEmpty()}
+                    options={chartConfig(pageState)} 
+                   /> 
+                   :
+                   <Bar
+                   data={chartData(testState)}
+                   options={chartConfig(pageState)}
+                 />}
               </div>
             </div>
+            <div className="columns" >
+                <div className="column" style={{width:'300px',paddingLeft:'0px'}}>
+                    <div className="column " style={{width:"200px"}}>
+                        <ProgressBar></ProgressBar>
+                    </div>
+                </div>
+            </div>
           </div>
           <StressPanel params={testParams}></StressPanel>
         </div>
         <div className="columns">
           <div className="column is-one-fifth">
           <div className="control">
             <input className="input" type="text" placeholder="Iterations" onInput={(e) => {
+                    resetChart();
```

Comment:
I'm not sure we want to reset the chart when the number of iterations changes, only when we press the run button for a new test